src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx

```diff
@@ -81,7 +81,7 @@ New Relic's time frames for supported regulatory frameworks and annual audits in
 
     <tr>
       <td>
-        APM (plus language agents)
+        [APM](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#apm-endpoints) (plus language agents)
```

Comment:
Why not call this "APM language agents"? The "plus language agents" seems confusing to me because that is all you have in APM, right?

Reply:
yeah that's weird I don't get why that was like that.